templates/web/fixmystreet.com/about/privacy.html

```diff
@@ -73,50 +73,71 @@
 
 <h3>Measuring website usage (Google Analytics)</h3>
 
-<p>We use Google Analytics to collect information about how people use this
-site. We do this to make sure it’s meeting its users’ needs and to understand
-how we could do it better. Google Analytics stores information such as what
-pages you visit, how long you are on the site, how you got here, what you click
-on, and information about your web browser. IP addresses are masked (only a
-portion is stored) and personal information is only reported in aggregate. We
-do not allow Google to use or share our analytics data for any purpose besides
-providing us with analytics information, and we recommend that any user of
-Google Analytics does the same.
-
-<p>If you’re unhappy with data about your visit to be used in this way, you can
-install the <a href="http://tools.google.com/dlpage/gaoptout">official browser
-plugin for blocking Google Analytics</a>.
-
-<p>The cookies set by Google Analytics are as follows:
-
-<table cellpadding=5>
-<tr align="left"><th scope="col">Name</th><th scope="col">Typical Content</th><th scope="col">Expires</th></tr>
-<tr><td>_ga</td><td>Used to distinguish users</td><td>2 years</td></tr>
-<tr><td>_gat</td><d>Used to throtle request rate</td><td>10 minutes</td></tr>
-<tr><td>__utmx / __utmxx</td><td>Which variation of a page you are seeing if we are testing different versions to see which is best</td><td>2&nbsp;years</td></tr>
-</table>
-
-<h4>Google’s Official Statement about Analytics Data</h4>
-
-<p>“This website uses Google Analytics, a web analytics service provided by
-Google, Inc. (“Google”). Google Analytics uses “cookies”, which are text files
-placed on your computer, to help the website analyze how users use the site.
-The information generated by the cookie about your use of the website
-(including your IP address) will be transmitted to and stored by Google on
-servers in the United States . Google will use this information for the purpose
-of evaluating your use of the website, compiling reports on website activity
-for website operators and providing other services relating to website activity
-and internet usage. Google may also transfer this information to third parties
-where required to do so by law, or where such third parties process the
-information on Google’s behalf. Google will not associate your IP address with
-any other data held by Google. You may refuse the use of cookies by selecting
-the appropriate settings on your browser, however please note that if you do
-this you may not be able to use the full functionality of this website. By
-using this website, you consent to the processing of data about you by Google
-in the manner and for the purposes set out above.”</p>
-
-<p><a href="https://www.mysociety.org/privacy-online/">More general information
-on how third party services work</a></p>
+<p>We use Google Analytics to collect information about how people
+use this site. We do this to make sure it&rsquo;s meeting its
```

Comment:
Could all these entities be the actual characters, please. And ideally, match the previous line width wherever possible so this diff is as small as necessary (that last part more for e.g. the Official Statement as I imagine that hasn't changed much, if at all).